econml/dml.py

```diff
@@ -130,18 +146,31 @@ def const_marginal_effect(self, X=None):
             X = np.ones((1, 1))
         return self._model_final.predict(X)
 
-    # need to override effect in case of discrete treatments
+    def const_marginal_effect_interval(self, X=None, *, alpha=0.1):
+        if X is None:
+            X = np.ones((1, 1))
+        return super()._const_marginal_effect_interval(X, alpha=alpha)
```

Comment:
Where is _const_marginal_effect_interval implemented? i.e. with the underscore in the beginning?

Reply:
Good catch, this was a bug.  I've added a test for it and fixed it.